### ISSUE TITLE
Previous declaration error when compiling openpilot after updating to new NEOS

### ISSUE THREAD

**Issue by energee** [commaai/eon-neos#1]
```
make: *** [Makefile:86: freethedsp.o] Error 1
make: *** Waiting for unfinished jobs....
building ('selfdrive/orbd', ['./orbd']) failed, make clean
rm -f *.o turbocv.so orbd test/turbocv_profile test/turbocv_test test/*.o *_lut.h                                                                               
[ CXX ] orbd_dsp.o
clang++ -std=c++11 -g -fPIC -O2  -Iinclude -I../../phonelibs/json/src -I. \
        -I../../phonelibs/capnp-cpp/include \
        -I../../phonelibs/zmq/aarch64/include \
        -I../../phonelibs/opencv/include \
        -DDSP \
        -I../ \
        -I../../ \
        -I../../../ \
        -I./include \
        -c -o 'orbd_dsp.o' 'orbd.cc'
[ CC ] calculator_stub.o
[ CC ] freethedsp.o
clang -std=gnu11 -g -fPIC -O2  -Iinclude -I../../phonelibs/json/src -I. -I./include -c -o 'calculator_stub.o' 'dsp/gen/calculator_stub.c'                       
clang -std=gnu11 -g -fPIC -O2  -Iinclude -I../../phonelibs/json/src -I. \
        -c -o 'freethedsp.o' 'dsp/freethedsp.c'
dsp/freethedsp.c:70:5: error: conflicting types for 'ioctl'
int ioctl(int fd, unsigned long request, void *arg) {
    ^
/data/data/com.termux/files/usr/include/bits/ioctl.h:36:5: note: previous declaration is here                                                                   
int ioctl(int __fd, int __request, ...);
    ^
1 error generated.
```

**Comment by geohot** [comment]
Reproduced. Fixed in openpilot 0.4.6

**Comment by energee** [comment]
There are other broken references, but I assume these are fixed, too.
